types/result.go

```diff
@@ -76,10 +76,11 @@ type TxResponse struct {
 	Tags      StringTags      `json:"tags,omitempty"`
 	Codespace string          `json:"codespace,omitempty"`
 	Tx        Tx              `json:"tx,omitempty"`
+	Timestamp string          `json:"timestamp,omitempty"`
 }
 
 // NewResponseResultTx returns a TxResponse given a ResultTx from tendermint
-func NewResponseResultTx(res *ctypes.ResultTx, tx Tx) TxResponse {
+func NewResponseResultTx(res *ctypes.ResultTx, tx Tx, timestamp string) TxResponse {
```

Comment:
++

Reply:
Can we get a test case for this? It would give a tiny boost to our coverage report

Reply:
A test case for the constructor?

Reply:
I know, it sounds dumb but it'd be easy and we would reduce the impact on the coverage report